mw-webapp/src/component/usersBlock/UsersBlock.tsx

```diff
@@ -17,6 +17,7 @@ export const UsersBlock = () => {
   }, []);
 
   const renderUsers = () => {
+    const users = Object.values(usersList);
```

Comment:
what does this line do? you are trying to get array from array with the same values or i miss siimething?

Reply:
It''s because of on fb collections is objects with key type string. And when we mapping this data we need firstly convert it to array where key will be 0,1,2... If you know how to map objects with any keys tell us

Reply:
If the keys in the database are sequential - 0, 1, 2...n, then we get an array at once and there are no errors, otherwise - an object, and on the object map does not work.
At first, I thought that the error was due to the fact that the key I use as userId contains letters. So I tried first removing all letters and leaving only numbers before using userId as a key. But that didn't work because the resulting number was large and still returned an object
Then I tried to get the length of the users collection from the database to assign the key based on the number of users. This didn't work because I couldn't get the length of the collection before writing a new user to the database
In the end, I decided to use Object.values to get an array of user data.